### PR TITLE
Feature/about

### PR DESCRIPTION
**Pull request by AIbrahimQA**

### CODE REVIEW

File: app/controllers/HomeController.scala [hunk 1]
```diff
@@ -11,9 +11,19 @@ class HomeController @Inject()(cc: ControllerComponents, val mongoService: Mongo
     Ok(views.html.index())
   }
 
+
+  def about: Action[AnyContent] = Action {
```
Comment (DannyPennington) [inline app/controllers/HomeController.scala]:
```suggestion
  def about: Action[AnyContent] = Action { implicit request:Request[AnyContent] =>
```

Comment (DannyPennington) [reply app/controllers/HomeController.scala]: implicit request required for header to compile

File: app/views/about.scala.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
+@()
```
Comment (DannyPennington) [inline app/views/about.scala.html]:
```suggestion
@()(implicit request: Request[AnyContent])
```

**Review by DannyPennington — CHANGES_REQUESTED** [review]
Looks good, just needs these changes to compile correctly

**Review by DannyPennington — APPROVED** [review]

**Review by JoNB7777 — APPROVED** [review]

**Pull request merged**